apps/gp2-frontend/src/users/export.ts

```diff
@@ -20,7 +20,8 @@ export const userFields = {
   workingGroups: 'Working groups',
   fundingStreams: 'Funding streams',
   contributingCohorts: 'Contributing cohorts',
-  createdDate: 'Activated account',
+  createdDate: 'Account Created',
+  activatedDate: 'Activated account',
```

Comment:
shouldn't it be Account Activated?

Reply:
It's what we had, but I think it should change.